src/app/etl-api/patient-program-resource.service.ts

```diff
@@ -19,7 +21,14 @@ export class PatientProgramResourceService {
   public getPatientProgramVisitConfigs(patientUuid: string): Observable<any> {
     let url = this.appSettingsService.getEtlRestbaseurl().trim();
     url += 'patient-program-config?patientUuid=' + patientUuid;
-    return this.http.get(url);
+    return this.http.get(url).pipe(retryWhen((error) => {
+      return error.pipe(flatMap((_error: any) => {
```

Comment:
Avoid using underscores. This is only allowed in get setters

Reply:
done